simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/handlers/ocpp/RequestStopTransactionRequestHandler.java

```diff
@@ -0,0 +1,46 @@
+package com.evbox.everon.ocpp.simulator.station.handlers.ocpp;
+
+import com.evbox.everon.ocpp.simulator.station.StationMessageSender;
+import com.evbox.everon.ocpp.simulator.station.StationState;
+import com.evbox.everon.ocpp.simulator.station.evse.Evse;
+import com.evbox.everon.ocpp.v20.message.station.RequestStopTransactionRequest;
+import com.evbox.everon.ocpp.v20.message.station.RequestStopTransactionResponse;
+import com.evbox.everon.ocpp.v20.message.station.TransactionData;
+import lombok.extern.slf4j.Slf4j;
+
+import static com.evbox.everon.ocpp.v20.message.station.TransactionEventRequest.TriggerReason.REMOTE_STOP;
+
+/**
+ * Handler for {@link RequestStopTransactionRequest} request.
+ */
+@Slf4j
+public class RequestStopTransactionRequestHandler implements OcppRequestHandler<RequestStopTransactionRequest> {
+
+    private final StationMessageSender stationMessageSender;
+    private final StationState stationState;
+
+    public RequestStopTransactionRequestHandler(StationMessageSender stationMessageSender, StationState stationState) {
+        this.stationMessageSender = stationMessageSender;
+        this.stationState = stationState;
+    }
+
+    /**
+     * Handle {@link RequestStopTransactionRequest} request.
+     *
+     * @param callId identity of the message
+     * @param request incoming request from the server
+     */
+    @Override
+    public void handle(String callId, RequestStopTransactionRequest request) {
+        stationMessageSender.sendCallResult(callId, new RequestStopTransactionResponse().withStatus(RequestStopTransactionResponse.Status.ACCEPTED));
```

Comment:
it will work in a wrong way when there is no such transaction. In this case you should respond with Rejected.